src/index.d.ts

```diff
@@ -16,7 +16,7 @@ export interface IOptions {
 
 // props specific to the React wrapper
 export interface SignatureCanvasProps extends IOptions {
-  canvasProps?: any,
+  canvasProps?: React.CanvasHTMLAttributes<object>,
```

Comment:
should the generic type be `<object>`?

Reply:
or maybe leave it as a generic as `<T>?: React.CanvasHTMLAttributes<T>`? I'm not totally sure why it's a generic or best practices around them in TS

Reply:
Pretty sure it should be a generic as that's referencing not the object containing the HTML attributes, but the attributes themselves which can be of various types, giving the object various potential shapes